docs/business-apps/power-automate/get-started/sharepoint-actions-triggers.md

```diff
@@ -201,19 +201,19 @@ Returns files in the root SharePoint folder.
 
 ### Move file
 
-Moves a file. Works similarly to the "Move to" command in SharePoint libraries. Returns info about the new file after its move.
+Moves a file. Works similarly to the **Move to** command in SharePoint libraries. Returns info about the new file after its move.
 
 ### Move folder
 
-Moves a folder. Works similarly to the "Move to" command in SharePoint libraries. Returns info about the new folder after its move.
+Moves a folder. Works similarly to the **Move to** command in SharePoint libraries. Returns info about the new folder after its move.
 
 ### Resolve person
 
 Returns a single matching user value so it can be assigned to a column of type person. If there are no matches, or multiple matches, this action errors out.
 
 ### Send an HTTP request to SharePoint
 
-Constructs a SharePoint REST API to invoke. Note: This action may execute any SharePoint REST API you have access to. Proceed with caution.
+Constructs a SharePoint REST API to invoke. **Note**: This action may execute any SharePoint REST API you have access to. Proceed with caution.
 
 ### Set content approval status
```

Comment:
Missing description: We can add the following:

_Sets the content approval status for an item in a list or library that has content approval turned on. You must provide an ETag for pages and files. You can get the ETag using the Get File Metadata action. This action is only available for SharePoint Online and SharePoint 2019._